x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/routes/disable.test.ts

```diff
@@ -15,6 +15,29 @@ import {
   requestMock,
 } from '../../../detection_engine/routes/__mocks__';
 import { riskEngineDataClientMock } from '../risk_engine_data_client.mock';
+import { securityMock } from '@kbn/security-plugin/server/mocks';
+
+export const createMockSecurityStartWithFullRiskEngineAccess = () => {
```

Comment:
I have gone a bit further and put the whole security mock in a mock file. I forgot .mock.ts existed I tried doing this with .test.ts but hat doesnt work because there must be a test in the file. Maybe I've gone too far but I use the same code in two places and itsn ot pretty which is why hiding it in a common util was appealing :D 

https://github.com/elastic/kibana/pull/172932/commits/533eaa774809c292934a986b791953490546ead5